pkg/operator/controller/credentials-secret/controller_test.go

```diff
@@ -124,14 +160,23 @@ func TestReconcile(t *testing.T) {
 	}
 
 	for _, tc := range testCases {
+
 		t.Run(tc.name, func(t *testing.T) {
 			cl := fake.NewClientBuilder().WithScheme(testScheme).WithRuntimeObjects(tc.existingObjects...).Build()
+			kubeFakeClient := kubernetesFakeClient.NewSimpleClientset()
+
+			if tc.name == "Bootstrap OCP" {
```

Comment:
Tests should not have the same name. Instead there could be a bool flag indicating if the fake resources should be loaded.

Reply:
@arjunrn  done